com.unity.multiplayer.mlapi/Tests/Runtime/MultiInstance/MultiInstanceHelpers.cs

```diff
@@ -170,6 +204,59 @@ public static IEnumerator WaitForClientConnected(NetworkManager client, Coroutin
             }
         }
 
+        /// <summary>
+        /// Similar to WaitForClientConnected, this waits for multiple clients to be connected.
+        /// </summary>
+        /// <param name="clients">The clients to be connected</param>
+        /// <param name="result">The result. If null, it will automatically assert<</param>
+        /// <param name="maxFrames">The max frames to wait for</param>
+        /// <returns></returns>
+        public static IEnumerator WaitForClientsConnected(NetworkManager[] clients, CoroutineResultWrapper<bool> result = null, int maxFrames = 64)
```

Comment:
This seems like a lot of duplicated logic, isn't there a way to reuse the other WaitForClientsConnected methods?

Reply:
Good point, I just made some adjustments which will remove the duplicate code while still keeping the methods in-tact.
( _Albin was the original author so I will leave it up to him to determine if we should completely remove the "wrapper methods"_)